x-pack/plugins/secops/.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Auto-collapse generated files in GitHub
+# https://help.github.com/en/articles/customizing-how-changed-files-appear-on-github
+x-pack/plugins/secops/public/graphql/types.ts linguist-generated=true
+x-pack/plugins/secops/server/graphql/types.ts linguist-generated=true
+x-pack/plugins/secops/public/graphql/introspection.json linguist-generated=true
```

Comment:
FYI: It may take a little time for these changes to take effect as per https://github.com/github/linguist/issues/3737#issuecomment-317981064

Reply:
Also, thoughts on adding snapshots as well?

``` gitattributes
*.test.tsx.snap linguist-generated=true
```

Reply:
Oh this is NICE! Thanks